### ISSUE TITLE
Reactor field placed within vizy block gets duplicated when block is dragged

### ISSUE THREAD

**Issue by piotrpog** [verbb/vizy#15]
Before dragging:

![image](https://user-images.githubusercontent.com/42622545/117358989-1e910200-aeb7-11eb-8c56-51425973fe65.png)

After dragging:

![image](https://user-images.githubusercontent.com/42622545/117359046-2d77b480-aeb7-11eb-995f-dc3167880952.png)

**Comment by engram-design** [comment]
Similar to https://github.com/verbb/vizy/issues/12 there's some serious incompatibility issues with Redactor, and looking into what we can do.

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.5](https://github.com/verbb/vizy/releases/tag/1.0.5)
